.github/run-mpyl/action.yaml

```diff
@@ -7,7 +8,7 @@ inputs:
     required: true
   command:
     description: The command to run
-    required: true
+    required: false
```

Comment:
Caught by the linter: this cannot be true as the `health` group [has no command](https://github.com/Vandebron/github-actions/pull/247/files#diff-29a44eb580fdb19157a276678765d30dcb4a8abdc9cade847b149e9bd2d094dcR128-R131).

Reply:
Makes sense, `command` used to be mandatory until I split it into `group` and `command`